### ISSUE TITLE
About the training process of the Quality Assessment Network

### ISSUE THREAD

**Issue by Wenretium** [kalpeshjp89/USISResNet#5]
I used your codes of model and trained on the KADID10k dataset, which is split into about 4:1 respectively for training and validation. However, I discover that the training loss kept decreasing while the validation loss kept in about 1(both used MSE as criterion). Here is the relation curve of them. Red stands for training and blue stands for validation. Train for 50 epochs at all.
![Loss](https://user-images.githubusercontent.com/55617704/109451164-c50bf180-7a87-11eb-807a-fa0578533180.jpg)
I want to know more about your traning details of this part, such as your optimizer, learning rate. I will appreciate it if you're willing to help me. 
Plus, my email is: wenretium@163.com.

**Comment by kalpeshjp89** [comment]
I replied on your mail ID about it.
Thank you.

And feel free to ask if required.
